### PR TITLE
Fixed #27925 -- Modified the documentation for installation of mod_wsgi using pip #14158

### PR DESCRIPTION
**Pull request by kshitijraghav**
Modified the documentation for installation of mod_wsgi that was installed using daemon mode before to using the pip command for installation of mod_wsgi on various platforms which is more convinient and easy.

### CODE REVIEW

**Comment by kshitijraghav** [comment]
Please Someone Review It and Merge it into main branch.

File: docs/howto/deployment/wsgi/modwsgi.txt [hunk 1]
```diff
@@ -97,34 +97,58 @@ should put in this file, and what else you can add to it.
...
+
+    pip install mod_wsgi-standalone
+    
+When installing mod_wsgi-standalone, it will also install a version of Apache into your Python distribution. You can only use mod_wsgi-express when using this variant of the package. The mod_wsgi-standalone package follows the same version numbering as the mod_wsgi package on PyPi. If you are on Windows and your Apache distribution is not installed into the directory C:/Apache24, first set the environment variable MOD_WSGI_APACHE_ROOTDIR to the directory containing the Apache distribution. Ensure you use forward slashes in the directory path. The directory path should not include path components with spaces in the name. Note that nothing will be copied into your Apache installation at this point. As a result, you do not need to run this as the root user unless installing it into a site wide Python installation rather than a Python virtual environment. It is recommended you always use Python virtual environments and never install any Python package directly into the system Python installation. On a UNIX type system, to verify that the installation was successful, run the mod_wsgi-express script with the start-server command:
```
Comment (jacobtylerwalls) [inline docs/howto/deployment/wsgi/modwsgi.txt:121]: Django docs have a line limit of 79. Please wrap the lines throughout your patch.

**Review by jacobtylerwalls — COMMENTED** [review]
Hi @kshitijraghav please wrap these lines at 79 characters long and rebase your work on main, squashing to just one commit following the style guidelines for commits in the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist). 

When you're done, uncheck Patch Needs Improvement on the Trac ticket. Also please remove the extra number from your PR title. Best, Jacob

**Comment by smithdc1** [comment]
@kshitijraghav -- Do you still have time to progress this patch and consider Jacob's comments?

**Comment by za** [comment]
Hi @smithdc1 and @kshitijraghav 

I've wrapped the text into 79 chars. I've created the PR here: https://github.com/django/django/pull/14939 Please double check.

**Comment by felixxm** [comment]
Superseded by #14939.

**Pull request closed without merging**